src/components/Book.js

```diff
@@ -22,12 +22,28 @@ const Book = (props) => {
         <p className="gender">{category}</p>
         <p className="title">{title}</p>
         <p className="author">{author}</p>
+        <ul className="listItems">
+          <li><a href="#top" onClick={foo}>Comments</a></li>
+          <li><a href="#top" id={id} onClick={handleRemoveBook}>Remove</a></li>
+          <li><a href="#top" onClick={foo}>Edit</a></li>
+        </ul>
+      </div>
+      <div className="completed">
+        <div className="progressContainer">
+          <div className="progress" />
+        </div>
+        <div>
+          <div>
+            <h1 className="value">75%</h1>
+            <p className="completedText">Completed</p>
+          </div>
+        </div>
+      </div>
+      <div>
+        <p className="curent-chapter">Current chapter</p>
+        <p className="chapter">Chapter 10</p>
+        <input type="submit" value="Update Progress" />
```

Comment:
- Great job so far in following the required design. There are some issues to work on before you can move on.
- [ ] Kindly use the correct color on your nav links and increase the font-weight of your `Bookstore cms`
- [ ] Also, when a nav link is active it has a different color. Kindly fix that.
- Your design:
![Annotation 2022-05-13 184158](https://user-images.githubusercontent.com/85113567/168341784-96d0dee4-7522-43c4-8923-f513473af316.jpg)
- Required design:
![Annotation 2022-05-13 184141](https://user-images.githubusercontent.com/85113567/168341813-e6af6c5a-dafd-405b-99aa-00e6f97a7daa.jpg)

- Next for the book cards, there are a number of issues to be taken care of.
- [ ] First your `update progress` button should not have any border and it should be longer.
- [ ] Next, there is a vertical line dividing the percentage and current chapter section. Kindly add that.
- [ ] There are also 2 vertical lines dividing the `comment`, `edit` and `remove` texts. It would be good to add that.
- [ ] Your `current chapter` text should be aligned with your `Not defined` text as in the design.
- Your design:
![Annotation 2022-05-13 184253](https://user-images.githubusercontent.com/85113567/168342454-4cb4ecea-babe-4a9f-a072-0d9f9e432030.jpg)
- Required design:
![Annotation 2022-05-13 184307](https://user-images.githubusercontent.com/85113567/168342497-582c87cc-9082-44ae-8a15-a6ac195f906c.jpg)

- Finally, there are a few issues to resolve in the form.
- [ ] The `add book` button should be longer and should not have any border. Also, apply the correct font to the button text and align the button correctly 👍 
- [ ] Kindly apply the correct color and font weight to the `Add new book` heading. Also, align the heading properly as shown in the design.
- Your design:
![Annotation 2022-05-13 184530](https://user-images.githubusercontent.com/85113567/168343078-3f4ac576-c04c-47cf-93b9-f49352d6702c.jpg)
- Required design:
![Annotation 2022-05-13 184513](https://user-images.githubusercontent.com/85113567/168343120-8f96d0a0-1feb-4b96-b09c-7a35028983d4.jpg)

Reply:
Thanks, @Kingstalux, I fixed the most of the reported issues, but I have to tell you that at this moment there is nos access to [Zeplin](https://app.zeplin.io/login) so I submitted a request to Zendesk and they send me this link to follow the style: [Reference site](https://pedantic-beaver-cb923b.netlify.app/).